notebooks/data_selection.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    telescope_pointing = SkyCoord(ra=dcheck_runsummary['mean_ra']*u.deg, dec=dcheck_runsummary['mean_dec']*u.deg)
</pre><p>Shouldn't we do this first? It will probably sort out much more runs than the other checks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
It is already the first thing, except for a fast check on problems with interleaveds. Nothing of this (except the loading of files, which may take O(1min) and TBH did not become noticeably faster) takes any significant amount of time.